Cargo.toml

```diff
@@ -18,9 +18,12 @@ zeroize = { version = "1.4", features = ["zeroize_derive"] }
 # k256 baggage
 k256 = { version = "0.9", features = ["ecdsa", "zeroize"], default-features = false }
 ecdsa = { version = "0.12", features = ["digest"], default-features = false } # needed only for FromDigest trait
+ed25519 = { version = ">=2.2, <2.3", features = ["pkcs8"], default-features = false }
+ed25519-dalek = { git = "https://github.com/dalek-cryptography/ed25519-dalek", tag = "2.0.0-rc.3", features = ["alloc", "digest", "rand_core"], default_features = false }
 rand = "0.8"
 sha2 = { version = "0.9", features = ["std", "asm"], default-features = false }
 sha3 = { version = "0.9", default-features = false }
+x509-ocsp = { git = "https://github.com/RustCrypto/formats", commit = "6e60929b2903b64d4d4fd5d3f2e705680352519d" }
```

Comment:
You could just use the `der` crate and copy the relevant struct, perhaps sans the `certs` field unless you actually intend to use that. Otherwise it's safe to ignore since it's `OPTIONAL`.

Reply:
Sure, that works for me. Will push an update soon.